laspy/__init__.py

```diff
@@ -1,6 +1,6 @@
 from __future__ import absolute_import
 
-__version__ = '1.6.0'
+__version__ = '1.7.0'
```

Comment:
Note I plan to roll this functionality out with a new release. Also included will be something related to #102